nntrainer/layers/acti_func.cpp

```diff
@@ -34,6 +34,9 @@ int ActiFunc::setActivation(
   std::function<Tensor &(Tensor const &, Tensor &)> const &activation_fn,
   std::function<Tensor &(Tensor &, Tensor &, Tensor const &)> const
     &activation_prime_fn) {
+  if (in_place)
```

Comment:
This is for the softmax kind ? and it raise the error if it is in_place?

Reply:
Yes, this is for softmax kind. Yes, softmax does not support in-place. This is a quick fix.
This check will later move to finalize with layer_v2. I will do this once layer_v2 merges with main.